mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -234,6 +242,9 @@ namespace mbf_abstract_nav
     //! The time the controller responded with a success output (output < 10).
     ros::Time last_valid_cmd_time_;
 
+    //! The time when the robot started ignoring velocity commands
+    ros::Time first_ignored_time_;
```

Comment:
F: Please rename this variable. Right now it's called `first_ignored_time_` but it's actually storing the last time a command was NOT ignored. A better name would be something like `last_followed_cmd_time_` or something similar

Reply:
It is actually storing the first time that the cmd_vel is being ignored by the robot. It will be set zero when the robot doesn't ignore cmd_vel. It keeps track of the first time robot ignored the cmd_vel, and it will be used to calculate the ignore_duration when the robot kept ignoring the cmd_vel until tolerance exceeded. Once the robot no longer ignores the cmd_vel, it is being set to zero again.

Reply:
Ah, I see. I thought you were always calling `ros::Time::now()` but you're actually calling `ros::Time()` and thus resetting to zero. Seems ok then!